src/Engine/Graphics/DecalBuilder.h

```diff
@@ -97,8 +97,8 @@ struct DecalBuilder {
         RenderVertexSoft* faceverts, char uClipFlags);
     bool ApplyBloodsplatDecals_IndoorFace(unsigned int uFaceID);
     bool ApplyBloodSplat_OutdoorFace(ODMFace* pFace);
-    bool ApplyBloodSplatToTerrain(struct Polygon* a2, Vec3f* a3, float* a4,
-                                  struct RenderVertexSoft* a5, unsigned int uStripType, char a7);
+    bool ApplyBloodSplatToTerrain(struct Polygon* terrpoly, Vec3f* terrnorm, float* tridotdist,
```

Comment:
Would be nice to get a doxydoc for this method.

Reply:
And please add a TODO to turn the magic numbers for `uStripType` and `tri_orient` into enums.